tasks/options/axe-webdriver.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function( grunt ) {
+grunt.loadNpmTasks("axe-grunt-webdriver");
+return {
+	firefox: {
+		options: {
+			threshold: 0
+		},
+		urls: ['http://localhost:4200/demos/typography.html'],
+		dest: 'tmp/gu.json'
+	},
+	chrome: {
```

Comment:
The travis will crash otherwise. We should have a separate test that uses chrome, that test will not be a part of travis.